### ISSUE TITLE
Stateless component with HOC expected syntax

### ISSUE THREAD

**Issue by aweber1** [reactjs/react-docgen#129]
Hi,

I have a stateless component wrapped in a HOC written like this:
```
import React, { PropTypes } from 'react';
import extendStyles from 'enhancers/extendStyles';

const Test = props =>
  <div style={props.style}>
    Hello world!
  </div>;

Test.propTypes = {
  style: PropTypes.object,
};

export default extendStyles(Test);

```

Unfortunately, when I try to parse it I get `Error: No suitable component definition found.`

If I remove the HOC, the component is parsed.

My HOC looks like this:
```
import React, { PropTypes } from 'react';
import deepExtend from 'deep-extend';

export default (WrappedComponent, styles = {}) => {
  const Enhancer = (props) => {
    const computedStyles =
      typeof styles === 'function' ?
      styles(props) :
      styles;
    const style = deepExtend({}, computedStyles, props.style);
    return <WrappedComponent {...props} style={style} />;
  };

  Enhancer.propTypes = {
    style: PropTypes.object,
  };

  Enhancer.defaultProps = {
    style: {},
  };

  Enhancer.displayName =
    WrappedComponent.displayName ||
    WrappedComponent.name ||
    'Component';

  return Enhancer;
};
```

I'm not sure how I can change the code so that docgen will parse it. I'm open to making changes in the HOC, but would rather not change the way the component is written if possible.

Sorry for the code dump, but hoping it will help in repro or someone will see something obvious.

**Comment by fkling** [comment]
This is a know limitation and has been addressed before ( #80 ) and has been implemented in  #124 , we just haven't released a new version yet. I can do that now though. I'll update the thread when I did that.

**Comment by aweber1** [comment]
Ah! I did see so that issue and the PR before I posted, just thought it had already been released and that I was doing something unexpected :) Thanks!
